### ISSUE TITLE
Can't see eICR generated 

### ISSUE THREAD

**Issue by Hkhanal253** [drajer-health/eCRNow#832]
Scenario :
Using eCR Now 3.1
I am able to make successful patient Launch. The logs from ecrnow show that it is able to find encounter in FHIR server and complete the launch.  There are bsa bundle queries Json files generated that contain the patient encounter info retrieved from FHIR. Scheduled tasks are being created but seem to get rescheduled every time after execution time. We are using HISP Direct to transport eICR to aims platform but have not yet tested end to end connection with HISP  which we have scheduled for near future.

My question is, before getting hooked up for end to end testing with HISP,  I wanted to confirm if eICR are being generated. Interestingly I never see eICR generated from ecrnow app after patient launch. Nothing in eICR table neither in bsa output directory. Also debug is turned on to keep eICR files. The trigger test case scenario was created based on instruction from ecrnow GitHub Repo. I am wondering what I am missing to not see eICR. Ecrnow log doesn't show any error either.

**Comment by Hkhanal253** [comment]
Yes. I did follow the documentation and activated ersd from ECR specifications/ KAR configuration section and also enabled subscription. Output format is set to CDA_R11 like shown in documentation for COVID-19 encounter.

**Comment by nbashyam** [comment]
Is this still an issue ?

**Comment by Hkhanal253** [comment]
This has been resolved after updating coding system. Thanks
